unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/Tests/ChannelsFeatureFlagServiceShould.cs

```diff
@@ -0,0 +1,61 @@
+using NSubstitute;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace DCL.Chat.Channels
+{
+    public class ChannelsFeatureFlagServiceShould
+    {
+        private ChannelsFeatureFlagService service;
+        private DataStore dataStore;
+        private IUserProfileBridge userProfileBridge;
+
+        [SetUp]
+        public void SetUp()
+        {
+            dataStore = new DataStore();
+            userProfileBridge = Substitute.For<IUserProfileBridge>();
+            var ownUserProfile = ScriptableObject.CreateInstance<UserProfile>();
+            ownUserProfile.UpdateData(new UserProfileModel
+            {
+                userId = "ownId"
+            });
+            userProfileBridge.GetOwn().Returns(ownUserProfile);
+            service = new ChannelsFeatureFlagService(dataStore, userProfileBridge);
```

Comment:
A TearDown method could be added with the Dispose of this service

Reply:
done!